src/artrace.py

```diff
@@ -2354,3 +2355,37 @@ def phys_to_pix(array, pixlocn, axis):
     else:
         pixarr = arcytrace.phys_to_pix(array, diff)
     return pixarr
+
+def slit_image(slf, det, scitrace, obj, tilts=None):
```

Comment:
`slit_image()` appears to return an image where each pixel represents the distance from the central trace in the "spatial" direction. If this is the correct interpretation, then I'm not sure this routine calculates exactly that. I think this routine is correct if the tilts are represented by a straight line, but not if the tilts are curved. In other words, I think you want to calculate the distance to the central trace of the object, along constant wavelength.

However, this doesn't necessarily mean that anything needs to be changed here. In most cases, the tilts will be close to a straight line near the pixels where the object and background regions are defined.

Just in case someone in the future wants a more precise calculation, it would be good to add a comment in the docstring along these lines, if this is correct.

or... perhaps I have missed something?